src/app/content/utils/archiveTreeUtils.ts

```diff
@@ -137,10 +137,11 @@ export const archiveTreeSectionIsUnit = (section: LinkedArchiveTreeNode) =>
   isArchiveTree(section)
   && !!section.parent
   && archiveTreeSectionIsBook(section.parent)
-  && getArchiveTreeSectionNumber(section) === null
+  && section.contents.every(isArchiveTree)
 ;
 export const archiveTreeSectionIsChapter = (section: LinkedArchiveTreeNode): section is LinkedArchiveTree =>
   isLinkedArchiveTree(section)
   && !archiveTreeSectionIsBook(section)
   && getArchiveTreeSectionNumber(section) !== null
```

Comment:
could we replace this with a check to see if the parent is a unit or a book? see thread https://openstax.slack.com/archives/CH5514N57/p1600276301142100?thread_ts=1599832840.080900&cid=CH5514N57

Reply:
In this case "Answer Key" from [Amerivan Gov](https://rex-web.herokuapp.com/books/american-government-2e/pages/chapter-1) will be treated as a chapter and will be displayed in the StudyGuides Chapter Filter

We could do this change but then we should change filters for study guides filters - and probably also for highlights filters
This will also have an impact on `<title>` tags

Should I do this?

Reply:
no, we need to make sure answer key doesn't show up as a chapter. in that example i'm not sure how to differentiate it.

lets keep the logic you have now and maybe i'll work with content to put some type flags in the tree so we don't have to gues